charts/camunda-bpm-platform/templates/deployment.yaml

```diff
@@ -14,9 +14,14 @@ spec:
       {{- include "camunda-bpm-platform.selectorLabels" . | nindent 6 }}
   template:
     metadata:
-      {{- with .Values.podAnnotations }}
+      {{- if or (eq .Values.recreatePods true) (.Values.podAnnotations) }}
```

Comment:
We can change this from `(eq .Values.recreatePods true)` to `(.Values.recreatePods)` since it's a bool already.